api/test/unit/policy.test.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
```

Comment:
This doesn't test policy but policies.js

Move to /test/unit/routes/policies.test.js